Sources/FaunaDB/Query.swift

```diff
@@ -26,6 +26,104 @@ public struct Ref: Fn {
 
 // MARK: Basic Forms
 
+/// Returns a native reference to classes. This allows for example,
+/// paginate over all classes in a database.
+///
+/// [Reference](https://fauna.com/documentation/queries#basic_forms).
+public struct Classes: Fn {
+
+    var call: Fn.Call
+
+    /// - Parameter scope: the scope database or nothing for the current database.
+    public init(_ scope: Expr = NullV()) {
```

Comment:
I think `scope` should not be nameless here or on the functions bellow. It's a good hint.